src/BalanceLaws/prog_prim_conversion.jl

```diff
@@ -43,3 +43,21 @@ function primitive_to_prognostic!(bl, prog::Vars, prim::Vars, aux)
     prog_arr = parent(prog)
     prog_arr .= prim_arr
 end
+
+
+"""
+    construct_face_auxiliary_state!(bl::AtmosModel, aux_face::AbstractArray, aux_cell::AbstractArray, Δz::FT)
+Default constructor
+ - `bl` balance law
+ - `aux_face` face auxiliary variables to be constructed 
+ - `aux_cell` cell center auxiliary variable
+ - `Δz` cell vertical size 
+"""
+function construct_face_auxiliary_state!(
+    bl,
+    aux_face::AbstractArray,
+    aux_cell::AbstractArray,
+    Δz::FT,
+) where {FT}
+    aux_face .= aux_cell
```

Comment:
Is this correct?

Reply:
@Zhengyu-Huang ?

Reply:
it was in the original branch (that was rebased badly) 
https://github.com/CliMA/ClimateMachine.jl/blob/c21e3dc998775b5462b5313d0ab6754204e38f10/src/BalanceLaws/prog_prim_conversion.jl#L56

Reply:
@charleskawczynski : This is the default function(users define the function for different applications ), the face aux is used for flux computation. 
For the dry atmosphere, the only thing we need is the geopotential
For the moist atmosphere, we need both geopotential and temperature?
For other applications, we do not need aux ?

Reply:
Maybe we should just have a function stub so that it fails by default?

Actually, now that I'm seeing the implementation in AtmosModel, it looks like the only "safe" variable is `orientation.Φ`

Reply:
I’m going to remove the fallback

Reply:
It turns out it can't be removed because a bunch of other experiments use it. That's a bit concerning to me. I'll leave it in and open an issue.